react-nd/src/components/Topbar/Topbar.jsx

```diff
@@ -0,0 +1,55 @@
+import styles from "./Topbar.module.scss";
+import Logo from "../../assets/logo.svg";
+import { LoginButton } from "../LoginButton/LoginButton";
+import { ROUTES } from "../../router/consts";
+import { Link } from "react-router-dom";
+import { useContext } from "react";
+import { UserContext } from "../../contexts/UserContext";
+import { Avatar } from "../Avatar/Avatar";
+
+export function Topbar() {
+  const { user, logout } = useContext(UserContext);
+
+  const links = [
```

Comment:
Kintamieji/funkcijos kurie nenaudoja steate/props turėtų būti aprašomi virš komponeneto arba atskirame faile.
Jei to nedarote - jie persikuria kiekvieną kart persikrovus komponentui iš švaistomi resursai